components/cdc/src/delegate.rs

```diff
@@ -555,11 +580,18 @@ impl Delegate {
             Ok(())
         };
 
-        let mut rows: HashMap<Vec<u8>, EventRow> = HashMap::default();
+        let mut txn_rows: HashMap<Vec<u8>, EventRow> = HashMap::default();
+        let mut raw_rows: HashMap<Vec<u8>, EventRow> = HashMap::default();
```

Comment:
RawKV do not need the HashMap. In txn, `commit_ts` is in `write_cf`, while value is in `defaut_cf`, so the HashMap is necessary to connect the two entry.

Reply:
So IMO, `sink_data` between txn & raw is quite different. It would be better to totally isolate these two. I would recommand by generic. By functions would also be OK.

Reply:
RawKV also need HashMap. When a key named "rkey1" is added to b, then “rkey1” may also appear. we just need the last one “rkey1”.